go/pkg/cas/client.go

```diff
@@ -271,6 +271,7 @@ func (c *Client) init() {
 	c.semByteStreamWrite = semaphore.NewWeighted(int64(c.Config.ByteStreamWrite.Concurrency))
 
 	c.semFileIO = semaphore.NewWeighted(int64(c.Config.FSConcurrency))
+	c.semLargeFile = semaphore.NewWeighted(2)
```

Comment:
Should this also be configurable with default 2?

Reply:
done